### DISCUSSION TITLE
Dupliquer

### DISCUSSION THREAD

**Discussion by Despotikgirl** [skarab42/marv#40] [Ideas]
Salut !
Est ce que ça serais possible de dupliquer ou copier des bouton ou sources qu'on à déjà créer sur marv?

**Comment by skarab42** [comment]
C'est prévu ainsi que #30 et #31

**Comment by skarab42** [comment]
C'est implémenter pour la prochaine release ;)

**Comment by skarab42** [comment]
Disponible dans la v1.4.0 ;)
https://marv-site.web.app/en/download
